### ISSUE TITLE
Loinc 27365-6:  "Kappa light chains [Mass/volume] in Urine"

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3647]
Suggest creating a new term for Loinc: 27365-6 [Kappa light chains [Mass/volume] in Urine] 
New term label:
New term comment (if any):
Your biocurator ID for loinc2hpo (if desired): HPO:probinson

**Comment by pnrobinson** [comment]
This should be near https://hpo.jax.org/app/browse/term/HP:0030156

**Comment by pnrobinson** [comment]
Note there are  2 classes of Bence Jones proteins designated kappa and lambda.

**Comment by pnrobinson** [comment]
added as HP_0032302

**Comment by kingmanzhang** [comment]
done
